dwpost/dwpost.php

```diff
@@ -8,177 +8,184 @@
+
+	/* provide a submit button */
+	$s .= '<div class="settings-submit-wrapper" ><input type="submit" id="dwpost-submit" name="dwpost-submit" class="settings-submit" value="' . L10n::t('Save Settings') . '" /></div></div>';
 }
 
 
-function dwpost_settings_post(&$a,&$b) {
-
-	if(x($_POST,'dwpost-submit')) {
-
-		PConfig::set(local_user(),'dwpost','post',intval($_POST['dwpost']));
-		PConfig::set(local_user(),'dwpost','post_by_default',intval($_POST['dw_bydefault']));
-		PConfig::set(local_user(),'dwpost','dw_username',trim($_POST['dw_username']));
-		PConfig::set(local_user(),'dwpost','dw_password',trim($_POST['dw_password']));
-
+function dwpost_settings_post(App $a, array &$b)
+{
+	if (!empty($_POST['dwpost-submit'])) {
+		PConfig::set(local_user(), 'dwpost', 'post',            intval($_POST['dwpost']));
+		PConfig::set(local_user(), 'dwpost', 'post_by_default', intval($_POST['dw_bydefault']));
+		PConfig::set(local_user(), 'dwpost', 'dw_username',     trim($_POST['dw_username']));
+		PConfig::set(local_user(), 'dwpost', 'dw_password',     trim($_POST['dw_password']));
 	}
-
 }
 
-function dwpost_post_local(&$a,&$b) {
-
+function dwpost_post_local(App $a, array &$b)
+{
 	// This can probably be changed to allow editing by pointing to a different API endpoint
-
-	if($b['edit'])
+	if ($b['edit']) {
 		return;
+	}
 
-	if((! local_user()) || (local_user() != $b['uid']))
+	if ((!local_user()) || (local_user() != $b['uid'])) {
 		return;
+	}
 
-	if($b['private'] || $b['parent'])
+	if ($b['private'] || $b['parent']) {
 		return;
+	}
 
-    $dw_post   = intval(PConfig::get(local_user(),'dwpost','post'));
+	$dw_post = intval(PConfig::get(local_user(),'dwpost','post'));
 
 	$dw_enable = (($dw_post && x($_REQUEST,'dwpost_enable')) ? intval($_REQUEST['dwpost_enable']) : 0);
 
-	if($_REQUEST['api_source'] && intval(PConfig::get(local_user(),'dwpost','post_by_default')))
+	if ($_REQUEST['api_source'] && intval(PConfig::get(local_user(),'dwpost','post_by_default'))) {
 		$dw_enable = 1;
+	}
 
-    if(! $dw_enable)
-       return;
-
-    if(strlen($b['postopts']))
-       $b['postopts'] .= ',';
-     $b['postopts'] .= 'dwpost';
-}
-
-
-
+	if (!$dw_enable) {
+		return;
+	}
 
-function dwpost_send(&$a,&$b) {
+	if (strlen($b['postopts'])) {
+		$b['postopts'] .= ',';
+	}
 
-    if($b['deleted'] || $b['private'] || ($b['created'] !== $b['edited']))
-        return;
+	$b['postopts'] .= 'dwpost';
+}
 
-    if(! strstr($b['postopts'],'dwpost'))
-        return;
+function dwpost_send(App $a, array &$b)
+{
+	if ($b['deleted'] || $b['private'] || ($b['created'] !== $b['edited'])) {
+		return;
+	}
 
-    if($b['parent'] != $b['id'])
-        return;
+	if (!strstr($b['postopts'],'dwpost')) {
+		return;
+	}
 
-	// dreamwidth post in the LJ user's timezone.
-	// Hopefully the person's Friendica account
-	// will be set to the same thing.
+	if ($b['parent'] != $b['id']) {
+		return;
+	}
 
+	/*
+	 * dreamwidth post in the LJ user's timezone.
```

Comment:
I think this LJ should be a DW in the comment.

Reply:
What have I overseen?

Reply:
Nothing serious. A leftover from the creation of the addon in the comments I think. Looks liky Tony made the LJ addon and then based the DW addon on it and that LJ in the comment above is a hint to that.

Reply:
Ah okay.